Wrappers/Python/cil/io/TIFF.py

```diff
@@ -29,40 +29,69 @@
 import glob
 import re
 import numpy as np
+from cil.io import utilities
 
 import logging
 
 logger = logging.getLogger(__name__)
 
+def compress_data(data, scale, offset, dtype):
+    '''Compress data to dtype using scale and offset
+    
+    Parameters
+    ----------
+    data : numpy array
+    scale : float
+    offset : float
+    dtype : numpy dtype
+    
+    returns compressed casted data'''
+    if dtype == data.dtype:
+        return data
+    if data.ndim > 2:
+        # compress each slice
+        tmp = np.empty(data.shape, dtype=dtype)
+        for i in range(data.shape[0]):
+            tmp[i] = compress_data(data[i], scale, offset, dtype)
+    else:
+        tmp = data * scale + offset
+        tmp = tmp.astype(dtype)
+    return tmp
+
 class TIFFWriter(object):
     '''Write a DataSet to disk as a TIFF file or stack'''
     
     def __init__(self,
                  **kwargs):
         '''
-        :param data: the data to save to TIFF file(s)
-        :type data: DataContainer, AcquisitionData or ImageData
-        :param file_name: string defining the file name prefix
-        :type file_name: string
-        :param counter_offset: int indicating at which number the index starts. 
+
+        Parameters
```

Comment:
This isn't in keeping with out docstrings

Here's an example:
https://github.com/TomographicImaging/CIL/blob/d642d8cb27c64ca3c3ae2b5c49f226365e2656ba/Wrappers/Python/cil/recon/FBP.py#L374-L404

it should be:
```
name : type, default
    description
```

If the description is too long I would add it to Notes, and add an Example.